Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml

```diff
@@ -74,6 +79,48 @@
       recoilFlat: 2
       scatterFlat: 6
 
+- type: entity
+  parent: RMCStockAttachmentBase
+  id: RMCAttachmentMOU53Stock
+  name: MOU53 Stock
+  description: A metal stock fitted specifically for the MOU53 break action shotgun.
+  components:
+  - type: Sprite
+    state: ou_stock
+  - type: AttachableVisuals
+    suffix: ""
+  - type: Tag
+    tags:
+    - RMCAttachmentStock
+    - RMCAttachmentMOU53Stock
+  - type: AttachableWeaponMeleeMods
+    modifiers:
+    - bonusDamage:
+        types:
+          Blunt: 10
+  - type: AttachableSpeedMods
+    modifiers:
+    - conditions:
+        wieldedOnly: true
+      walk: -0.050
+      sprint: -0.10
+  - type: AttachableWieldDelayMods
+    modifiers:
+      - delay: 0.3
+  - type: AttachableWeaponRangedMods
+    modifiers:
+    - conditions:
+        wieldedOnly: true
+      accuracyAddMult: 0.3
+      recoilFlat: -4
+      scatterFlat: -5
+    - conditions:
+        unwieldedOnly: true
+      accuracyMovementPenaltyAddMult: -1
+      accuracyAddMult: -0.1
+      recoilFlat: 2
+      scatterFlat: 5
```

Comment:
These modifiers aren't correct, scatter wielded is -1 in 13 which should be -2 here, check AttachableData.cs file for conversions

Reply:
Adjusted scatter values